### ISSUE TITLE
V3 Beta10: Default PSModulePath on Linux includes non-existent paths causing Get-PsResource to fail

### ISSUE THREAD

**Issue by ChrisAtRelativity** [PowerShell/PSResourceGet#255]
# Reproduce

On a Debian 10 (Buster) distro:

* Install Powershell 7
* Install-Module Az
* Install-Module PowershellGet -AllowPrerelease -Force
* Register-PsResourceRepository <a repo>
* Get-PsResource Az
* The second path in the default PSModulePath does not exist and Get-PsResource fails
* After creating the path '/usr/local/share/powershell/Modules', Get-PsResource _still_ fails

# Error

```powershell
PS /home/jenkins> Get-PsResource Az
Get-PSResource: Could not find a part of the path '/root/.local/share/powershell/Modules:/usr/local/share/powershell/Modules:/opt/microsoft/powershell/7/Modules'.
PS /home/jenkins> $env:PSModulePath
/root/.local/share/powershell/Modules:/usr/local/share/powershell/Modules:/opt/microsoft/powershell/7/Modules
PS /home/jenkins> ls /usr/local/share/powershell/
Scripts
```

# Workaround

Setting $env:PSModulePath to a single path allows Get-PsResource to work

```powershell
$env:PSModulePath = '/root/.local/share/powershell/Modules'
```

**Comment by SydneyhSmith** [comment]
Thanks for providing this info @ChrisAtRelativity we will fix this in the 3.0 time frame

**Comment by alerickson** [comment]
This will be resolved with the release of Preview 11.
